app/views/shared/check_answers/_payments_summary.html.erb

```diff
@@ -3,35 +3,59 @@
 <% individual = partner ? "Partner" : "Applicant"
    regular_payments = partner || @legal_aid_application.client_uploading_bank_statements?
    url = payments_summary_partial_url(partner, credit, regular_payments, @legal_aid_application) %>
+
+<% individual_with_determiner = partner ? t("generic.partner") : t("generic.client") %>
+
 <% if credit
      payment_type = t(".credits_section_heading.#{individual.downcase}")
      transaction_types = @credit_transaction_types
+     none_heading = t(".credits_section_heading.none", individual_with_determiner:)
    else
      payment_type = t(".debits_section_heading.#{individual.downcase}")
      transaction_types = @debit_transaction_types
+     none_heading = t(".debits_section_heading.none", individual_with_determiner:)
    end %>
 
+<% transactions = if @legal_aid_application.client_uploading_bank_statements? || individual.eql?("Partner")
+                    if credit
+                      @legal_aid_application.regular_transactions.where(owner_type: individual).credits.without_housing_benefits.without_disregarded_benefits.without_benefits
+                    else
+                      @legal_aid_application.regular_transactions.where(owner_type: individual).debits
+                    end
+                  else
+                    transaction_types.map { |t|
+                      answer_for_transaction_type(legal_aid_application: @legal_aid_application, transaction_type: t, owner_type: individual)
+                    }.reject { |value| value == "None" }
+                  end %>
```

Comment:
There may well be a better way to do all this

Reply:
I'm working on it here: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/7764